README.md

```diff
@@ -83,6 +90,34 @@ This step creates the following defaults:
 2. component files in the components folder called `pubsub.yaml`, `statestore.yaml` and `zipkin.yaml`. 
 3. default config file `$HOME/.dapr/config.yaml` for Linux/MacOS or for Windows at `%USERPROFILE%\.dapr\config.yaml` to enable tracing on `dapr init` call. Can be overridden with the `--config` flag on `dapr run`.
 
+#### Without Docker 
```

Comment:
same as above

Reply:
For this one, since we are using --slim, we should use that term, something like "Slim init" or "Minimal installation"